sox.features.js

```diff
@@ -81,22 +81,19 @@
 
         copyCommentsLink: function() {
```

Comment:
Were changes to this function by accident? (It seems to have rolled back my changes)

Reply:
Probably. I do recall starting work on the latest dev branch. Ah...you went from `dev*.*.14` to `dev*.*.15` by the time I finished work, Hadn't noticed that. Sorry! Feel free to roll back to your original.